### PR TITLE
adds shakespeare sample test

### PR DESCRIPTION
**Pull request by bshaffer**
To satisfy the simple example requirements

### CODE REVIEW

File: bigquery/shakespeare/phpunit.xml.dist [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<phpunit bootstrap="test/bootstrap.php">
+    <testsuites>
+        <testsuite name="PHP bigquery test">
```
Comment (tmatsuo) [inline bigquery/shakespeare/phpunit.xml.dist]: worth adding `shakespeare` to the testsuite name?

File: bigquery/shakespeare/phpunit.xml.dist [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            <directory>test</directory>
+        </testsuite>
+    </testsuites>
+    <logging>
+        <log type="coverage-clover" target="build/logs/clover.xml"/>
+    </logging>
+    <filter>
+        <whitelist>
+            <file>bigquery.php</file>
```
Comment (tmatsuo) [inline bigquery/shakespeare/phpunit.xml.dist]: `shakespeare.php`?

File: bigquery/shakespeare/phpunit.xml.dist (hunk 1 above)
Comment (bshaffer) [inline bigquery/shakespeare/phpunit.xml.dist]: yes!

File: bigquery/shakespeare/phpunit.xml.dist (hunk 2 above)
Comment (bshaffer) [inline bigquery/shakespeare/phpunit.xml.dist]: yes!

File: bigquery/shakespeare/phpunit.xml.dist [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+            <directory>test</directory>
+        </testsuite>
+    </testsuites>
+    <logging>
+        <log type="coverage-clover" target="build/logs/clover.xml"/>
+    </logging>
+    <filter>
+        <whitelist>
+            <file>shakespeare.php</file>
+        <directory suffix=".php">./src</directory>
```
Comment (tmatsuo) [inline bigquery/shakespeare/phpunit.xml.dist]: Nit: Indentation is off

**Pull request merged**